tracex_project/extraction/templates/choose_input_method.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Choose Input Method</title>
+    {% load static %}
+    <link rel="stylesheet" type="text/css" href="{% static '/tracex/css/style.css' %}">
+</head>
+
+<body class="main_body">
+    <img src="{% static '/tracex/img/extraction_logo.png' %}" alt="Extraction Logo" class="extraction_logo">
+    <h2>Choose Input Method</h2>
+    <p> The extraction pipeline helps you extract XES Event Logs from Patient Journeys. You can filter the results by event type and location before and after extraction to get exactly what you need. </p>
+    <p>Please select the patient journey you want to upload below.</p>
+    <div class="form-row">
+        <div class="form-column">
+            <a href='{% url 'journey_upload' %}'>
```

Comment:
u have to use " instead of ' 

see below in code for correct way: "{% url 'landing_page' %}"